Doc/tools/touch-clean-files.py

```diff
@@ -6,16 +6,27 @@
 
 from pathlib import Path
 
-# Input file has blank line between entries to reduce merge conflicts
-with Path("Doc/tools/clean-files.txt").open() as clean_files:
-    CLEAN = [
+# Exclude these whether they're dirty or clean,
+# because they trigger a rebuild of dirty files.
+EXCLUDES = {
+    Path("Doc/includes/wasm-notavail.rst"),
```

Comment:
Maybe just exclude the `include` directory as a whole, to exclude future `.rst` files added to it (which would have the same behavior)?

Reply:
Yep, will add `include` to an `EXCLUDE_SUBDIRS`